packages/terra-menu/src/MenuItem.module.scss

```diff
@@ -56,7 +56,7 @@
     }
 
     .start-icon {
-      color: var(--terra-menu-item-disabled-start-icon-color, #9b9fa1);;
+      color: var(--terra-menu-item-disabled-start-icon-color, #9b9fa1);
```

Comment:
I am not sure why this was not caught by style-lint. style lint should have caught this while building the code. if we don't have rule specified for extra-semicolon error then we might need to add rule 
``` "no-extra-semicolons":true, ``` in [stylelint-config-terra](https://github.com/cerner/stylelint-config-terra/blob/f2b8bc0f908df3ada150d077e2964dd6996063f4/stylelint.config.js#L14) to catch these (copy-paste) errors in earlier stage.

Reply:
@supreethmr Good suggestion. PR to add the `no-extra-semicolons` rule: https://github.com/cerner/stylelint-config-terra/pull/31